indico/htdocs/sass/modules/_event_management.scss

```diff
@@ -505,7 +501,7 @@ th.i-table {
         display: block;
         position: relative;
         background-color: $light-gray;
-        border-bottom: 1px solid $gray;
+        @include border-bottom();
```

Comment:
I think is necessary ;) Just created a wiki page, similar to the JS one: https://github.com/indico/indico/wiki/SASS-Guidelines